server/src/main/assets/stylesheets/main.less

```diff
@@ -20,6 +20,27 @@
 @colHighlight: #666666;
 @dropDown: false;
 
+html, body {
+  height: 100%;
+  margin: 0;
+}
+
+#root {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.main-content {
+  flex: 1;
+}
+
+footer {
```

Comment:
I don't think this is being used anywhere

Reply:
it is used in Layout with bottombar .